src/nupic/regions/RecordSensor.py

```diff
@@ -511,3 +519,37 @@ def setParameter(self, parameterName, index, parameterValue):
 
     else:
       raise Exception('Unknown parameter: ' + parameterName)
+
+
+  def write(self, proto):
+    """Write state to proto object.
+
+    proto: PyRegionProto capnproto object
+    """
+    regionImpl = proto.regionImpl.as_struct(RecordSensorProto)
+
+    self.encoder.write(regionImpl.encoder)
+    self.disabledEncoder.write(regionImpl.disabledEncoder)
+    regionImpl.topDownMode = self.topDownMode
+    regionImpl.verbosity = self.verbosity
+    regionImpl.numCategories = self.numCategories
+
+    # TODO: Write preEncodingFilters and postEncodingFilters
```

Comment:
Yeah probably for now. I'm doubt that functionlaity would worked if we tried to use it.